test/wasm2c/call-indirect-import.txt

```diff
@@ -0,0 +1,932 @@
+;;; TOOL: run-wasm2c
+(module
+  (type (;0;) (func (param i32)))
+  (type (;1;) (func))
+  (type (;2;) (func (result i32)))
+  (type (;3;) (func (param i32 i32) (result i32)))
+  (import "env" "__linear_memory" (memory (;0;) 0))
+  (import "env" "__indirect_function_table" (table (;0;) 1 funcref))
+  (func $invoke (type 0) (param i32)
+    local.get 0
+    call_indirect (type 1))
+  (func $callee (type 1))
+  (func $__original_main (type 2) (result i32)
+    i32.const 1
+    call $invoke
+    i32.const 0)
+  (func $main (type 3) (param i32 i32) (result i32)
+    call $__original_main)
+  (elem (;0;) (i32.const 1) func $callee))
```

Comment:
This looks like it was compiler-generated, but could be simplified quite a bit if it was more hand-written.

For example, I don't think the `__linear_memory` is used at all?   And you don't need the `__original_main` / `main` split you?  (can just put the body of `__original_main` into `main` and delete `__original_main`?)

Reply:
I had left the linear memory in, so we could see the codegen for that import as well. I can rename the text to `check-import` to make it clear we are importing both tables and memory. I'll remove the duplicate mains.

Reply:
Sounds good!